scripts/build/setup/termux_setup_gn.sh

```diff
@@ -0,0 +1,30 @@
+termux_setup_gn() {
+	termux_setup_ninja
+	local GN_COMMIT=eea3906f0e2a8d3622080127d2005ff214d51383
+	local GN_FOLDER=$TERMUX_COMMON_CACHEDIR/gn-$GN_COMMIT
+	local GN_TARFILE=$TERMUX_COMMON_CACHEDIR/gn_$GN_COMMIT.tar.gz
+	local GN_SOURCE=https://gn.googlesource.com/gn/+archive/$GN_COMMIT.tar.gz
+	test ! -f $GN_TARFILE && termux_download \
+		$GN_SOURCE \
+		$GN_TARFILE \
+		SKIP_CHECKSUM
+	mkdir -p $GN_FOLDER
+	tar xf $GN_TARFILE -C $GN_FOLDER
```

Comment:
You are extracting tar without checking the presence of already extracted installation.